samples/snippets/src/main/java/com/example/firestore/snippets/ManageDataSnippets.java

```diff
@@ -468,6 +473,58 @@ public void updateDocumentIncrement() throws ExecutionException, InterruptedExce
     updateFuture.get();
   }
 
+  /**
+   * Run a snapshot read to read past versions of documents.
+   *
+   * @return transaction future
+   */
+  DocumentSnapshot runSnapshotReads() throws Exception {
+    // [START firestore_snapshot_read]
+    // Initialize a documnet
+    final DocumentReference documentReference = db.collection("cities").document("SF");
+    City city = new City("SF");
+    city.setCountry("USA");
+    city.setPopulation(860008L);
+    // Write doc and save write result
+    WriteResult writeResult = documentReference.set(city).get();
+
+    // Create a reference to the cities collection
+    CollectionReference cities = db.collection("cities");
+    // Create a query against the collection.
+    Query query = cities.whereEqualTo("capital", true);
+
+    // In the transaction options, Set read time and and set to read-only
+    // Snapshot reads require read-only transactions
+    // As an example, set read time to the update time of the previous write operation
+    final Timestamp readTime = com.google.protobuf.Timestamp.newBuilder()
+        .setSeconds(writeResult.getUpdateTime().getSeconds())
+        .setNanos(writeResult.getUpdateTime().getNanos())
+        .build();
+    // final Timestamp readTime = writeResult.getUpdateTime();
```

Comment:
why do we have this commented line which returns the same timestamp as the above lines that constructs timestamp from the builder? is this for demonstration purpose?

Reply:
Left in that comment by mistake. Removed. Thanks!